app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+    get '/sessions/new' do
+	erb :"sessions/new.html"
+    end
+    post '/sessions' do
+	@user = User.find_by(email: params[:email])
+	@email = params[:email]
```

Comment:
Rather than storing the email and user as instance variables, you can pass them into the `erb` template via. `locals`